lib/react_phoenix/react_io.ex

```diff
@@ -1,5 +1,7 @@
 defmodule ReactPhoenix.ReactIo do
   @moduledoc false
 
-  use StdJsonIo, otp_app: :react_phoenix, script: Application.get_env(:react_phoenix, :react_stdio_path, ReactPhoenix.ReactIo.PathFinder.react_stdio_path())
+  use StdJsonIo,
+    otp_app: :react_phoenix,
+    script: (Application.get_env(:react_phoenix, :react_stdio_path) || ReactPhoenix.ReactIo.PathFinder.react_stdio_path())
```

Comment:
`Application.get_env/2` actually accepts a 3rd parameter. This parameter is the default value if the environment variable is not set. Thus, the original code is effectively the same as your changes. I'd like to keep the original usage of `Application.get_env/2` but keep your styling/formatting.